### DISCUSSION TITLE
What would I use for WIKI_DIR if the base-repo is the wiki?

### DISCUSSION THREAD

**Discussion by Andre601** [Andrew-Chen-Wang/github-wiki-action#15] [Q&A] [answered]
Right now updating a repository to have a dedicated, orphaned branch for the Wiki itself and would like to know what I need to set if the repo itself (root) is used as the `WIKI_DIR`

Would it be `.`?
Or `./`?

**Comment by Andrew-Chen-Wang** [comment]
There's two possibilities that you can try:
1. Use "."
2. Try "$PWD"

**Comment by jcbhmr** [comment, marked as answer]
For anyone who comes across this: With the release of [v4](https://github.com/Andrew-Chen-Wang/github-wiki-action/releases/tag/v4.0.0), the answer to this is:

```yml
- uses: Andrew-Chen-Wang/github-wiki-action@v4
  with:
    path: .
```
